ansible/roles/controller/tasks/deploy.yml

```diff
@@ -27,6 +27,14 @@
     mode: 0777
   become: "{{ logs.dir.become }}"
 
+# We need to create the file with proper permissions because the dir creation above
```

Comment:
take a look at the lines just below - maybe we can consolidate both?
i'm wondering if we should guard this for docker-machine env.
@csantanapr thoughts?